.github/workflows/publish.yml

```diff
@@ -0,0 +1,20 @@
+name: neptune-sacred
+on: [push]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Build and test
+        run : |
+          python setup.py sdist
+      - name: Publish package
+        uses: pypa/gh-action-pypi-publish@54b39fb9371c0b3a6f9f14bb8a67394defc7a806
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
You'll need to ask @HubertJaworski to generate a secret for a new PyPI package and set it as a repo secret to make the first release through tagging.

Reply:
Yes, but first I'll need to give him *.tar.gz file. I wanted to generate it after marging to master.

Reply:
Just saying!